__test__/index.test.js

```diff
@@ -0,0 +1,21 @@
+/* eslint-env jest */
+
+import responseToDataURI from '../src/index'
+import { dataURLToBlob } from 'blob-util'
+import 'whatwg-fetch'
```

Comment:
detail: I always sort the imports to first have external libs, then an empty line and then local imports. Might be a habit stemming from Python, but I found it a good habit.